acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/permissioning/AllowlistWithDnsPersistorAcceptanceTest.java

```diff
@@ -67,22 +71,48 @@ public void setUp() throws Exception {
     cluster.start(this.node);
   }
 
-  @Ignore("test is failing in CI")
   @Test
-  public void manipulatedNodesAllowlistWithHostnameShouldWorkWhenDnsEnabled() {
+  public void addingEnodeWithIp_andThenAddingSameEnodeWithHostname_shouldThrow() {
```

Comment:
What about the case where add with hostname first and then with IP? Do we have a test?